### DISCUSSION TITLE
ReadMe Update

### DISCUSSION THREAD

**Discussion by crogonint** [SlRvb/Obsidian--ITS-Theme#105] [Q&A] [answered]
Brand new to Obsidian, but I checked for a built in Plug-in repository before I ran off and side-loaded Style Settings. It's built in to the Community Plug-ins now. You might update your readme to mention that. :) EDIT: My Bad! That;s actually in your alternative...md, not the readme.md (I couldn't find the Alternative ITS in the list of themes.)

Also, in about two-days.. I'll be scratching my head wondering what the advantage was to releasing "Themes" this way. ..?

**Comment by SlRvb** [comment, marked as answer]
The snippet came first, then I had some people asking me to include it in my theme. The only advantage was just to try and save some space in kbs with the alternate colors and mbs with all the encoded fonts. 

Only benefit to the snippet version is Fonts are embedded so you don't have to install them and color schemes will apply while using another theme. The ITS Theme receives updates/fixes more frequently than the snippets will, so that's one thing to keep in mind.
